composer.json

```diff
@@ -23,8 +23,8 @@
         }
     ],
     "require": {
-        "illuminate/database": "^9.0",
-        "illuminate/support": "^9.0"
+        "illuminate/database": "^8.0|^9.0",
+        "illuminate/support": "^8.0|^9.0"
```

Comment:
No backward compatibility. I only support the current major version of Laravel.

Reply:
Sure, I've changed to what it was